docs/deployment/semgrep-app-ghe.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: semgrep-app-ghe
+append_help_link: true
+title: Install the Semgrep App for GitHub Enterprise
```

Comment:
Hmm, so - we have an app in the GitHub marketplace that is public. When we say "Install the Semgrep App for GHE" users might think we have a public app, but the process is to actually make a private app (IIRC). Do we have any alternative title ideas?

Reply:
Yeah, I was struggling with this a bit -- wdyt about "Install a private Semgrep App for GHE?"